### PR TITLE
Implement Get label by id endpoint

### PR DESCRIPTION
**Pull request by SuruiLiu**
Endpoint can return correct data.
488ad1da78f483f7509dab6d20551c8e

### CODE REVIEW

File: blotztask-api/BlotzTask.http [hunk 1]
```diff
@@ -4,3 +4,7 @@ GET {{BlotzTask_HostAddress}}/weatherforecast/
 Accept: application/json
 
 ###
+
+GET {{BlotzTask_HostAddress}}/api/label/1
```
Comment (sol-wizard) [inline blotztask-api/BlotzTask.http:8]: what is this file for? do you need them?

File: blotztask-api/Controllers/LabelController.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+        public LabelController(ILabelService labelService)
+        {
+            _labelService = labelService;
+        }
+
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetLabelById(int id)
+        {
+            return Ok(await _labelService.GetLabelById(id)); // Return 200 OK with the label data
```
Comment (sol-wizard) [inline blotztask-api/Controllers/LabelController.cs:23]: remove comment from gpt

File: blotztask-api/Controllers/LabelController.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿using BlotzTask.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BlotzTask.Controllers
+{
+    [Route("api/[controller]")]
```
Comment (sol-wizard) [inline blotztask-api/Controllers/LabelController.cs:8]: can you put this in the last order?

`    [Authorize]
    [ApiController]
    [Route("api/[controller]")]`

**Review by sol-wizard — CHANGES_REQUESTED** [review]

File: blotztask-api/BlotzTask.http (hunk 1 above)
Comment (SuruiLiu) [reply blotztask-api/BlotzTask.http:8]: This is a way to test the endpoint. Maybe we even don't need the file. 
105ecba3b6e4eeda002a9f576a0306df

**Review by sol-wizard — APPROVED** [review]

**Pull request merged**